### PR TITLE
CSL-31: PCA - Tell us about your main customers

### PR DESCRIPTION
**Pull request by dk4g**
## What? 
[CSL-31](https://collaboration.homeoffice.gov.uk/jira/browse/CSL-31) - 1.10 PCA - 'Tell us about your main customers' (The organisation and how it operates)
Added screens for questions:
- Tell us about your main customers
- Tell us about your main suppliers
- What security measures do you have in place at the premises?
- Describe how you secure your premises
## Anything Else? (optional)
Steps have been split into relevant sections and commented out to shorten the user journey to the summary page and speed up the testing process.
## Check list

- [ ] I have reviewed my own pull request
- [ ] I have written tests (if relevant)

### CODE REVIEW

File: apps/precursor-chemicals/translations/src/en/validation.json [hunk 1]
```diff
@@ -94,5 +94,23 @@
...
   "responsible-officer-dbs-update-subscription": {
       "required": "Select if the responsible officer has subscribed to DBS updates"
+  },
+  "main-customers": {
+    "required": "Enter details of your main customers",
+    "maxlength": "You have exceeded the 2000 character limit",
+    "notUrl": "Your answer must not contain URLs or links"
+  },
+  "main-suppliers": {
+    "required": "Enter details of your main suppliers"
```
Comment (vinodhasamiyappanHO) [inline apps/precursor-chemicals/translations/src/en/validation.json:104]: maxlength and notUrl has to be added for this field.

**Comment by RobertMcCann** [comment]
By commenting out  /upload-conduct-certificate it seems we remove the back button from /main-customers  - Shall we add /update-conduct-certificate or leave a note for QAT?

**Comment by dk4g** [comment]
> By commenting out /upload-conduct-certificate it seems we remove the back button from /main-customers - Shall we add /update-conduct-certificate or leave a note for QAT?

I have confirmed this with the QAT team and they are aware of the scope of the isolated 'The organisation and how it operates' feature branch. we will  uncomment all steps before merging back to the main branch, where the whole journey then will be tested in UAT.

**Review by RobertMcCann — APPROVED** [review]
Looks good to me!

**Pull request merged**